src/main/java/com/gooddata/UriPrefixingClientHttpRequestFactory.java

```diff
@@ -20,7 +20,7 @@
  * Sets default URI prefix for Spring REST client which by default requires absolute URI.
  * UriPrefixingAsyncClientHttpRequestFactory allows you to specify default hostname and port.
  */
-class UriPrefixingClientHttpRequestFactory implements ClientHttpRequestFactory {
+public class UriPrefixingClientHttpRequestFactory implements ClientHttpRequestFactory {
```

Comment:
i would rather leave this class as package private. can we find way around it in the internal library?